modules/rgbd/src/hash_tsdf.hpp

```diff
@@ -60,11 +61,30 @@ struct tsdf_hash
     }
 };
 
+
+//typedef std::unordered_map<cv::Vec3i, VolumeUnit, tsdf_hash> VolumeUnitMap;
+
+typedef unsigned int volumeIndex;
+struct VolumeUnit
+{
+    cv::Vec3i coord;
+    volumeIndex index;
+    cv::Matx44f pose;
+    Vec4i volDims;
+    bool isActive;
+};
+
 typedef std::unordered_set<cv::Vec3i, tsdf_hash> VolumeUnitIndexSet;
-typedef std::unordered_map<cv::Vec3i, VolumeUnit, tsdf_hash> VolumeUnitMap;
+typedef std::unordered_map<cv::Vec3i, VolumeUnit, tsdf_hash> VolumeUnitIndexes;
+//typedef std::unordered_set<cv::Vec3i> VolumeUnitIndexSet;
+//typedef std::unordered_map<cv::Vec3i, VolumeUnit> VolumeUnitIndexes;
 
 class HashTSDFVolumeCPU : public HashTSDFVolume
 {
+private:
+    void _integrate( cv::Matx44f _pose,  Point3i volResolution, Vec4i volDims,
```

Comment:
Please name it `integrateVolumeUnit` or somehow more self-descriptive than this

Reply:
done